From d43b4fbcb430cedab9463d70d11f587dbe10c8b2 Mon Sep 17 00:00:00 2001
From: cloud-comms-engine <engine@diegonmarcos.com>
Date: Tue, 11 Aug 2026 09:58:48 +0000
Subject: [PATCH 1/3] rebrand: Cloud Vault app_name, applicationId, launcher
 icon

- app_name -> "Cloud Vault" / "Cloud Vault Dev" / "Cloud Vault Beta" in the
  three buildType-scoped strings_non_localized.xml (main/beta/release).
- applicationId com.x8bit.bitwarden -> com.diegonmarcos.cloudvault.
- Replace the Bitwarden shield launcher mark with a simple original padlock
  mark (main + release + beta drawable overrides, since AGP merges the
  buildType-specific ic_launcher_foreground/monochrome over main's for the
  actual release build) and a new background color. No Bitwarden artwork is
  reused. Bitwarden trademarks are not used (see NOTICE, patch 3).
---
 app/build.gradle.kts                          |  2 +-
 .../res/drawable/ic_launcher_foreground.xml   | 42 +++++++------------
 .../res/drawable/ic_launcher_monochrome.xml   | 25 +++++++----
 .../beta/res/values/strings_non_localized.xml |  2 +-
 .../res/drawable/ic_launcher_foreground.xml   | 32 +++++---------
 .../res/drawable/ic_launcher_monochrome.xml   | 18 +++++---
 .../res/values/ic_launcher_background.xml     |  4 +-
 .../main/res/values/strings_non_localized.xml |  2 +-
 .../res/drawable/ic_launcher_foreground.xml   | 41 +++++++-----------
 .../res/drawable/ic_launcher_monochrome.xml   | 23 ++++++----
 .../res/values/strings_non_localized.xml      |  2 +-
 11 files changed, 92 insertions(+), 101 deletions(-)

diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index 9364dbc..901ac52 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -60,7 +60,7 @@ configure<ApplicationExtension> {
     }
 
     defaultConfig {
-        applicationId = "com.x8bit.bitwarden"
+        applicationId = "com.diegonmarcos.cloudvault"
         minSdk {
             version = release(libs.versions.minSdk.get().toInt())
         }
diff --git a/app/src/beta/res/drawable/ic_launcher_foreground.xml b/app/src/beta/res/drawable/ic_launcher_foreground.xml
index 91a12ef..c457a6b 100644
--- a/app/src/beta/res/drawable/ic_launcher_foreground.xml
+++ b/app/src/beta/res/drawable/ic_launcher_foreground.xml
@@ -1,36 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Cloud Vault mark (beta channel accent). Not derived from Bitwarden's shield artwork. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:aapt="http://schemas.android.com/aapt"
     android:width="108dp"
     android:height="108dp"
-    android:viewportHeight="108"
-    android:viewportWidth="108">
+    android:viewportWidth="108"
+    android:viewportHeight="108">
     <group>
-        <clip-path android:pathData="M0,0h108v108h-108z" />
-        <path android:pathData="M71.57,34.63L113,76.06L89.71,119.08L40.65,70.02L53.53,77.45L70.88,60.11L71.57,34.63Z">
-            <aapt:attr name="android:fillColor">
-                <gradient
-                    android:endX="89.01"
-                    android:endY="89.97"
-                    android:startX="37.85"
-                    android:startY="38.81"
-                    android:type="linear">
-                    <item
-                        android:color="#3F000000"
-                        android:offset="0" />
-                    <item
-                        android:color="#00000000"
-                        android:offset="1" />
-                </gradient>
-            </aapt:attr>
-        </path>
+        <clip-path
+            android:pathData="M0,0h108v108h-108z"/>
         <path
-            android:fillColor="#ffffff"
-            android:pathData="M71.51,34.56C71.15,34.19 70.71,34 70.21,34H36.85C36.35,34 35.92,34.19 35.56,34.56C35.19,34.92 35,35.35 35,35.85V58.09C35,59.75 35.33,61.4 35.97,63.03C36.62,64.66 37.42,66.11 38.38,67.37C39.33,68.64 40.47,69.87 41.8,71.06C43.12,72.26 44.35,73.25 45.47,74.04C46.59,74.83 47.76,75.58 48.97,76.28C50.19,76.98 51.06,77.46 51.57,77.72C52.09,77.97 52.49,78.16 52.8,78.29C53.03,78.41 53.29,78.47 53.55,78.47C53.82,78.47 54.08,78.41 54.31,78.29C54.61,78.15 55.03,77.97 55.54,77.72C56.05,77.47 56.91,76.98 58.13,76.28C59.35,75.58 60.52,74.82 61.64,74.04C62.76,73.25 63.98,72.26 65.31,71.06C66.63,69.86 67.77,68.63 68.73,67.37C69.68,66.1 70.48,64.65 71.13,63.03C71.77,61.39 72.1,59.75 72.1,58.09V35.85C72.07,35.35 71.88,34.92 71.51,34.56ZM67.21,58.3C67.21,66.35 53.53,73.29 53.53,73.29V38.76H67.21C67.21,38.76 67.21,50.25 67.21,58.3Z" />
+            android:pathData="M37,52 V42 A17,17 0 0 1 71,42 V52"
+            android:strokeColor="#FFFFFF"
+            android:strokeWidth="9"
+            android:strokeLineCap="round"
+            android:fillColor="#00000000"/>
         <path
-            android:fillColor="#6795E8"
-            android:pathData="M55,49C55,47.34 56.34,46 58,46H107C108.66,46 110,47.34 110,49V60C110,61.66 108.66,63 107,63H58C56.34,63 55,61.66 55,60V49Z" />
+            android:pathData="M32,50 H76 A8,8 0 0 1 84,58 V82 A8,8 0 0 1 76,90 H32 A8,8 0 0 1 24,82 V58 A8,8 0 0 1 32,50 Z"
+            android:fillColor="#FFFFFF"/>
         <path
-            android:fillColor="#212529"
-            android:pathData="M57.12,50.43H59.78C60.92,50.43 61.79,50.59 62.37,50.91C62.96,51.23 63.25,51.8 63.25,52.6C63.25,52.93 63.2,53.22 63.09,53.48C62.99,53.74 62.84,53.96 62.65,54.13C62.45,54.29 62.21,54.4 61.93,54.45V54.51C62.22,54.57 62.48,54.67 62.71,54.82C62.94,54.96 63.12,55.16 63.26,55.44C63.4,55.71 63.47,56.07 63.47,56.52C63.47,57.04 63.34,57.48 63.09,57.85C62.83,58.22 62.47,58.51 62,58.71C61.53,58.9 60.97,59 60.32,59H57.12V50.43ZM58.93,53.83H59.99C60.51,53.83 60.88,53.74 61.08,53.58C61.29,53.41 61.39,53.17 61.39,52.84C61.39,52.51 61.27,52.28 61.02,52.14C60.79,51.99 60.41,51.92 59.89,51.92H58.93V53.83ZM58.93,55.27V57.5H60.12C60.66,57.5 61.04,57.39 61.26,57.18C61.47,56.97 61.58,56.69 61.58,56.33C61.58,56.12 61.53,55.94 61.44,55.78C61.35,55.62 61.19,55.49 60.97,55.4C60.75,55.31 60.44,55.27 60.06,55.27H58.93ZM70.11,59H65.18V50.43H70.11V51.92H66.99V53.8H69.9V55.29H66.99V57.5H70.11V59ZM75.21,59H73.39V51.95H71.07V50.43H77.54V51.95H75.21V59ZM84.1,59L83.48,56.96H80.35L79.73,59H77.78L80.8,50.4H83.02L86.06,59H84.1ZM83.04,55.44L82.42,53.45C82.38,53.31 82.33,53.14 82.26,52.94C82.2,52.72 82.14,52.51 82.07,52.3C82.01,52.08 81.96,51.89 81.92,51.73C81.88,51.89 81.82,52.09 81.75,52.33C81.69,52.56 81.62,52.78 81.56,52.99C81.5,53.21 81.45,53.36 81.43,53.45L80.81,55.44H83.04Z" />
+            android:pathData="M54,61 C57.87,61 61,64.13 61,68 C61,70.71 59.43,73.06 57.15,74.24 L60,84 H48 L50.85,74.24 C48.57,73.06 47,70.71 47,68 C47,64.13 50.13,61 54,61 Z"
+            android:fillColor="#4FC3A1"/>
     </group>
 </vector>
diff --git a/app/src/beta/res/drawable/ic_launcher_monochrome.xml b/app/src/beta/res/drawable/ic_launcher_monochrome.xml
index 8dc8488..2315589 100644
--- a/app/src/beta/res/drawable/ic_launcher_monochrome.xml
+++ b/app/src/beta/res/drawable/ic_launcher_monochrome.xml
@@ -1,17 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Cloud Vault mark (monochrome/themed-icon variant, beta channel). -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="108dp"
     android:height="108dp"
-    android:viewportHeight="108"
-    android:viewportWidth="108">
+    android:viewportWidth="108"
+    android:viewportHeight="108">
     <group>
-        <clip-path android:pathData="M0,0h108v108h-108z" />
+        <clip-path
+            android:pathData="M0,0h108v108h-108z"/>
         <path
-            android:fillColor="#ffffff"
-            android:fillType="evenOdd"
-            android:pathData="M70.21,34C70.71,34 71.15,34.19 71.51,34.56C71.88,34.92 72.07,35.35 72.1,35.85V50.43H71.07V51.95H72.1V58.09C72.1,59.75 71.77,61.39 71.13,63.03C70.48,64.65 69.68,66.1 68.73,67.37C67.77,68.63 66.63,69.86 65.31,71.06C63.98,72.26 62.76,73.25 61.64,74.04C60.52,74.82 59.35,75.58 58.13,76.28C56.91,76.98 56.05,77.47 55.54,77.72C55.03,77.97 54.61,78.15 54.31,78.29C54.08,78.41 53.82,78.47 53.55,78.47C53.29,78.47 53.03,78.41 52.8,78.29C52.53,78.18 52.18,78.01 51.75,77.8C51.69,77.78 51.63,77.75 51.57,77.72C51.06,77.46 50.19,76.98 48.97,76.28C47.76,75.58 46.59,74.83 45.47,74.04C44.35,73.25 43.12,72.26 41.8,71.06C40.47,69.87 39.33,68.64 38.38,67.37C37.42,66.11 36.62,64.66 35.97,63.03C35.33,61.4 35,59.75 35,58.09V35.85C35,35.35 35.19,34.92 35.56,34.56C35.92,34.19 36.35,34 36.85,34H70.21ZM67.21,57.5H70.11V59H67.18C66.43,66.75 53.53,73.29 53.53,73.29V38.76H67.21V50.43H70.11V51.92H67.21V53.8H69.89V55.29H67.21V57.5Z" />
+            android:pathData="M37,52 V42 A17,17 0 0 1 71,42 V52"
+            android:strokeColor="#FF000000"
+            android:strokeWidth="9"
+            android:strokeLineCap="round"
+            android:fillColor="#00000000"/>
         <path
-            android:fillColor="#6795E8"
-            android:fillType="evenOdd"
-            android:pathData="M58,46C56.34,46 55,47.34 55,49V60C55,61.66 56.34,63 58,63H107C108.66,63 110,61.66 110,60V49C110,47.34 108.66,46 107,46H58ZM59.78,50.43H57.12V59H60.32C60.97,59 61.53,58.9 62,58.71C62.47,58.51 62.83,58.22 63.09,57.85C63.34,57.48 63.47,57.04 63.47,56.52C63.47,56.07 63.4,55.71 63.26,55.44C63.12,55.16 62.94,54.96 62.71,54.82C62.48,54.67 62.22,54.57 61.93,54.51V54.45C62.21,54.4 62.45,54.29 62.65,54.13C62.84,53.96 62.99,53.74 63.09,53.48C63.2,53.22 63.25,52.93 63.25,52.6C63.25,51.8 62.96,51.23 62.37,50.91C61.79,50.59 60.92,50.43 59.78,50.43ZM59.99,53.83H58.93V51.92H59.89C60.41,51.92 60.79,51.99 61.02,52.14C61.27,52.28 61.39,52.51 61.39,52.84C61.39,53.17 61.29,53.41 61.08,53.58C60.88,53.74 60.51,53.83 59.99,53.83ZM58.93,57.5V55.27H60.06C60.44,55.27 60.75,55.31 60.97,55.4C61.19,55.49 61.35,55.62 61.44,55.78C61.53,55.94 61.58,56.12 61.58,56.33C61.58,56.69 61.47,56.97 61.26,57.18C61.04,57.39 60.66,57.5 60.12,57.5H58.93ZM65.18,59H70.11V57.5H66.99V55.29H69.89V53.8H66.99V51.92H70.11V50.43H65.18V59ZM73.39,59H75.21V51.95H77.54V50.43H71.07V51.95H73.39V59ZM83.48,56.96L84.1,59H86.06L83.02,50.4H80.8L77.78,59H79.73L80.35,56.96H83.48ZM82.42,53.45L83.04,55.44H80.81L81.43,53.45C81.45,53.36 81.5,53.21 81.56,52.99C81.62,52.78 81.69,52.56 81.75,52.33C81.82,52.09 81.88,51.89 81.92,51.73C81.96,51.89 82.01,52.08 82.07,52.3C82.14,52.51 82.2,52.72 82.26,52.94C82.33,53.14 82.38,53.31 82.42,53.45Z" />
+            android:pathData="M32,50 H76 A8,8 0 0 1 84,58 V82 A8,8 0 0 1 76,90 H32 A8,8 0 0 1 24,82 V58 A8,8 0 0 1 32,50 Z"
+            android:fillColor="#FF000000"/>
+        <path
+            android:pathData="M54,61 C57.87,61 61,64.13 61,68 C61,70.71 59.43,73.06 57.15,74.24 L60,84 H48 L50.85,74.24 C48.57,73.06 47,70.71 47,68 C47,64.13 50.13,61 54,61 Z"
+            android:fillColor="#00000000"/>
     </group>
 </vector>
diff --git a/app/src/beta/res/values/strings_non_localized.xml b/app/src/beta/res/values/strings_non_localized.xml
index 326726c..0536e79 100644
--- a/app/src/beta/res/values/strings_non_localized.xml
+++ b/app/src/beta/res/values/strings_non_localized.xml
@@ -1,4 +1,4 @@
 <resources>
     <!--Common-->
-    <string name="app_name" translatable="false">Bitwarden Beta</string>
+    <string name="app_name" translatable="false">Cloud Vault Beta</string>
 </resources>
diff --git a/app/src/main/res/drawable/ic_launcher_foreground.xml b/app/src/main/res/drawable/ic_launcher_foreground.xml
index 8220d58..4ecd954 100644
--- a/app/src/main/res/drawable/ic_launcher_foreground.xml
+++ b/app/src/main/res/drawable/ic_launcher_foreground.xml
@@ -1,5 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Cloud Vault mark: a simple padlock. Not derived from Bitwarden's shield artwork. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:aapt="http://schemas.android.com/aapt"
     android:width="108dp"
     android:height="108dp"
     android:viewportWidth="108"
@@ -8,27 +9,16 @@
         <clip-path
             android:pathData="M0,0h108v108h-108z"/>
         <path
-            android:pathData="M71.57,34.63L113,76.06L89.71,119.08L40.65,70.02L53.53,77.45L70.88,60.11L71.57,34.63Z">
-            <aapt:attr name="android:fillColor">
-                <gradient
-                    android:startX="37.85"
-                    android:startY="38.81"
-                    android:endX="89.01"
-                    android:endY="89.97"
-                    android:type="linear">
-                    <item android:offset="0" android:color="#3F000000"/>
-                    <item android:offset="1" android:color="#00000000"/>
-                </gradient>
-            </aapt:attr>
-        </path>
+            android:pathData="M37,52 V42 A17,17 0 0 1 71,42 V52"
+            android:strokeColor="#FFFFFF"
+            android:strokeWidth="9"
+            android:strokeLineCap="round"
+            android:fillColor="#00000000"/>
         <path
-            android:pathData="M71.51,34.56C71.15,34.19 70.71,34 70.21,34H36.85C36.35,34 35.92,34.19 35.56,34.56C35.19,34.92 35,35.35 35,35.85V58.09C35,59.75 35.33,61.4 35.97,63.03C36.62,64.66 37.42,66.11 38.38,67.37C39.33,68.64 40.47,69.87 41.8,71.06C43.12,72.26 44.35,73.25 45.47,74.04C46.59,74.83 47.76,75.58 48.97,76.28C50.19,76.98 51.06,77.46 51.57,77.72C52.09,77.97 52.49,78.16 52.8,78.29C53.03,78.41 53.29,78.47 53.55,78.47C53.82,78.47 54.08,78.41 54.31,78.29C54.61,78.15 55.03,77.97 55.54,77.72C56.05,77.47 56.91,76.98 58.13,76.28C59.35,75.58 60.52,74.82 61.64,74.04C62.76,73.25 63.98,72.26 65.31,71.06C66.63,69.86 67.77,68.63 68.73,67.37C69.68,66.1 70.48,64.65 71.13,63.03C71.77,61.39 72.1,59.75 72.1,58.09V35.85C72.07,35.35 71.88,34.92 71.51,34.56ZM67.21,58.3C67.21,66.35 53.53,73.29 53.53,73.29V38.76H67.21C67.21,38.76 67.21,50.25 67.21,58.3Z"
-            android:fillColor="#ffffff"/>
+            android:pathData="M32,50 H76 A8,8 0 0 1 84,58 V82 A8,8 0 0 1 76,90 H32 A8,8 0 0 1 24,82 V58 A8,8 0 0 1 32,50 Z"
+            android:fillColor="#FFFFFF"/>
         <path
-            android:pathData="M55,49C55,47.34 56.34,46 58,46H107C108.66,46 110,47.34 110,49V60C110,61.66 108.66,63 107,63H58C56.34,63 55,61.66 55,60V49Z"
-            android:fillColor="#2DA49D"/>
-        <path
-            android:pathData="M64.22,54.63C64.22,55.6 64.04,56.41 63.67,57.05C63.3,57.7 62.77,58.19 62.07,58.51C61.37,58.84 60.53,59 59.54,59H57.12V50.43H59.81C60.7,50.43 61.48,50.59 62.14,50.91C62.8,51.23 63.32,51.7 63.68,52.33C64.04,52.95 64.22,53.72 64.22,54.63ZM62.34,54.68C62.34,54.05 62.24,53.53 62.06,53.12C61.87,52.71 61.6,52.41 61.24,52.21C60.88,52.02 60.43,51.92 59.9,51.92H58.93V57.5H59.71C60.6,57.5 61.26,57.26 61.69,56.79C62.12,56.32 62.34,55.62 62.34,54.68ZM70.93,59H66V50.43H70.93V51.92H67.82V53.8H70.72V55.29H67.82V57.5H70.93V59ZM79.45,50.43L76.53,59H74.55L71.65,50.43H73.48L75.09,55.53C75.12,55.62 75.16,55.78 75.23,56.02C75.29,56.25 75.35,56.5 75.41,56.77C75.48,57.03 75.53,57.25 75.55,57.42C75.57,57.25 75.61,57.03 75.67,56.77C75.74,56.51 75.8,56.26 75.85,56.02C75.92,55.79 75.96,55.62 75.99,55.53L77.61,50.43H79.45Z"
-            android:fillColor="#212529"/>
+            android:pathData="M54,61 C57.87,61 61,64.13 61,68 C61,70.71 59.43,73.06 57.15,74.24 L60,84 H48 L50.85,74.24 C48.57,73.06 47,70.71 47,68 C47,64.13 50.13,61 54,61 Z"
+            android:fillColor="#F2A93B"/>
     </group>
 </vector>
diff --git a/app/src/main/res/drawable/ic_launcher_monochrome.xml b/app/src/main/res/drawable/ic_launcher_monochrome.xml
index ca084f1..0e81584 100644
--- a/app/src/main/res/drawable/ic_launcher_monochrome.xml
+++ b/app/src/main/res/drawable/ic_launcher_monochrome.xml
@@ -1,3 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Cloud Vault mark (monochrome/themed-icon variant): same padlock silhouette. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="108dp"
     android:height="108dp"
@@ -7,12 +9,16 @@
         <clip-path
             android:pathData="M0,0h108v108h-108z"/>
         <path
-            android:pathData="M70.21,34C70.71,34 71.15,34.19 71.51,34.56C71.88,34.92 72.07,35.35 72.1,35.85V50.43H71.65L72.1,51.76V58.09C72.1,59.75 71.77,61.39 71.13,63.03C70.48,64.65 69.68,66.1 68.73,67.37C67.77,68.63 66.63,69.86 65.31,71.06C63.98,72.26 62.76,73.25 61.64,74.04C60.52,74.82 59.35,75.58 58.13,76.28C56.91,76.98 56.05,77.47 55.54,77.72C55.03,77.97 54.61,78.15 54.31,78.29C54.08,78.41 53.82,78.47 53.55,78.47C53.29,78.47 53.03,78.41 52.8,78.29C52.53,78.18 52.18,78.01 51.75,77.8C51.69,77.78 51.63,77.75 51.57,77.72C51.06,77.46 50.19,76.98 48.97,76.28C47.76,75.58 46.59,74.83 45.47,74.04C44.35,73.25 43.12,72.26 41.8,71.06C40.47,69.87 39.33,68.64 38.38,67.37C37.42,66.11 36.62,64.66 35.97,63.03C35.33,61.4 35,59.75 35,58.09V35.85C35,35.35 35.19,34.92 35.56,34.56C35.92,34.19 36.35,34 36.85,34H70.21ZM67.18,59C66.43,66.75 53.53,73.29 53.53,73.29V38.76H67.21V50.43H70.93V51.92H67.81V53.8H70.72V55.29H67.81V57.5H70.93V59H67.18Z"
-            android:fillColor="#ffffff"
-            android:fillType="evenOdd"/>
+            android:pathData="M37,52 V42 A17,17 0 0 1 71,42 V52"
+            android:strokeColor="#FF000000"
+            android:strokeWidth="9"
+            android:strokeLineCap="round"
+            android:fillColor="#00000000"/>
         <path
-            android:pathData="M58,46C56.34,46 55,47.34 55,49V60C55,61.66 56.34,63 58,63H107C108.66,63 110,61.66 110,60V49C110,47.34 108.66,46 107,46H58ZM63.67,57.05C64.04,56.41 64.22,55.6 64.22,54.63C64.22,53.72 64.04,52.95 63.68,52.33C63.31,51.7 62.8,51.23 62.14,50.91C61.48,50.59 60.7,50.43 59.81,50.43H57.12V59H59.54C60.53,59 61.37,58.84 62.07,58.51C62.77,58.19 63.3,57.7 63.67,57.05ZM62.06,53.12C62.24,53.53 62.34,54.05 62.34,54.68C62.34,55.62 62.12,56.32 61.69,56.79C61.26,57.26 60.6,57.5 59.71,57.5H58.93V51.92H59.9C60.43,51.92 60.88,52.02 61.23,52.21C61.6,52.41 61.87,52.71 62.06,53.12ZM66,59H70.93V57.5H67.81V55.29H70.72V53.8H67.81V51.92H70.93V50.43H66V59ZM76.53,59L79.45,50.43H77.61L75.99,55.53C75.96,55.62 75.92,55.79 75.85,56.02C75.8,56.26 75.73,56.51 75.67,56.77C75.61,57.03 75.57,57.25 75.55,57.42C75.53,57.25 75.48,57.03 75.41,56.77C75.35,56.5 75.29,56.25 75.23,56.02C75.16,55.78 75.12,55.62 75.09,55.53L73.48,50.43H71.65L74.55,59H76.53Z"
-            android:fillColor="#2DA49D"
-            android:fillType="evenOdd"/>
+            android:pathData="M32,50 H76 A8,8 0 0 1 84,58 V82 A8,8 0 0 1 76,90 H32 A8,8 0 0 1 24,82 V58 A8,8 0 0 1 32,50 Z"
+            android:fillColor="#FF000000"/>
+        <path
+            android:pathData="M54,61 C57.87,61 61,64.13 61,68 C61,70.71 59.43,73.06 57.15,74.24 L60,84 H48 L50.85,74.24 C48.57,73.06 47,70.71 47,68 C47,64.13 50.13,61 54,61 Z"
+            android:fillColor="#00000000"/>
     </group>
 </vector>
diff --git a/app/src/main/res/values/ic_launcher_background.xml b/app/src/main/res/values/ic_launcher_background.xml
index ddfad1d..b131d8c 100644
--- a/app/src/main/res/values/ic_launcher_background.xml
+++ b/app/src/main/res/values/ic_launcher_background.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- This is the apps Primary Blue color an exists here for launcher and splash theme only -->
-    <color name="ic_launcher_background">#FF175DDC</color>
+    <!-- Cloud Vault brand color (was Bitwarden's Primary Blue) - launcher and splash theme only -->
+    <color name="ic_launcher_background">#FF3E2472</color>
     <color name="ic_shortcut_background">@android:color/white</color>
 </resources>
diff --git a/app/src/main/res/values/strings_non_localized.xml b/app/src/main/res/values/strings_non_localized.xml
index 55c5c84..5574696 100644
--- a/app/src/main/res/values/strings_non_localized.xml
+++ b/app/src/main/res/values/strings_non_localized.xml
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name" translatable="false">Bitwarden Dev</string>
+    <string name="app_name" translatable="false">Cloud Vault Dev</string>
 </resources>
diff --git a/app/src/release/res/drawable/ic_launcher_foreground.xml b/app/src/release/res/drawable/ic_launcher_foreground.xml
index e4d87ca..4ecd954 100644
--- a/app/src/release/res/drawable/ic_launcher_foreground.xml
+++ b/app/src/release/res/drawable/ic_launcher_foreground.xml
@@ -1,33 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Cloud Vault mark: a simple padlock. Not derived from Bitwarden's shield artwork. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:aapt="http://schemas.android.com/aapt"
     android:width="108dp"
     android:height="108dp"
     android:viewportWidth="108"
     android:viewportHeight="108">
-    <group
-        android:scaleX="0.099"
-        android:scaleY="0.099"
-        android:translateX="24.3"
-        android:translateY="24.3">
-        <path android:pathData="M482,103l418,418l-235,434l-495,-495l130,75l175,-175z">
-            <aapt:attr name="android:fillColor">
-                <gradient
-                    android:endX="573.9931"
-                    android:endY="815.5847"
-                    android:startX="57.80515"
-                    android:startY="213.12973"
-                    android:type="linear">
-                    <item
-                        android:color="#3F000000"
-                        android:offset="0" />
-                    <item
-                        android:color="#00000000"
-                        android:offset="1" />
-                </gradient>
-            </aapt:attr>
-        </path>
+    <group>
+        <clip-path
+            android:pathData="M0,0h108v108h-108z"/>
         <path
-            android:fillColor="#ffffff"
-            android:pathData="M481.4,102.2c-3.7,-3.7 -8.1,-5.6 -13.1,-5.6L131.7,96.6c-5.1,0 -9.4,1.9 -13.1,5.6C114.9,105.9 113,110.2 113,115.3v224.4c0,16.7 3.3,33.4 9.8,49.8c6.5,16.5 14.6,31.1 24.3,43.8c9.6,12.8 21.1,25.2 34.5,37.2c13.3,12.1 25.7,22.1 37,30.1c11.3,8 23.1,15.5 35.4,22.6c12.3,7.1 21,11.9 26.2,14.5c5.2,2.5 9.3,4.5 12.4,5.8c2.3,1.2 4.9,1.8 7.6,1.8c2.7,0 5.3,-0.6 7.6,-1.8c3.1,-1.4 7.3,-3.3 12.4,-5.8c5.2,-2.5 13.9,-7.4 26.2,-14.5c12.3,-7.1 24.1,-14.7 35.4,-22.6c11.3,-8 23.6,-18 37,-30.1c13.3,-12.1 24.8,-24.5 34.5,-37.2c9.6,-12.8 17.7,-27.4 24.2,-43.8c6.5,-16.5 9.8,-33.1 9.8,-49.8L487.3,115.3C487,110.2 485.1,105.9 481.4,102.2zM438,341.8C438,423 300,493 300,493L300,144.6h138C438,144.6 438,260.6 438,341.8z" />
+            android:pathData="M37,52 V42 A17,17 0 0 1 71,42 V52"
+            android:strokeColor="#FFFFFF"
+            android:strokeWidth="9"
+            android:strokeLineCap="round"
+            android:fillColor="#00000000"/>
+        <path
+            android:pathData="M32,50 H76 A8,8 0 0 1 84,58 V82 A8,8 0 0 1 76,90 H32 A8,8 0 0 1 24,82 V58 A8,8 0 0 1 32,50 Z"
+            android:fillColor="#FFFFFF"/>
+        <path
+            android:pathData="M54,61 C57.87,61 61,64.13 61,68 C61,70.71 59.43,73.06 57.15,74.24 L60,84 H48 L50.85,74.24 C48.57,73.06 47,70.71 47,68 C47,64.13 50.13,61 54,61 Z"
+            android:fillColor="#F2A93B"/>
     </group>
 </vector>
diff --git a/app/src/release/res/drawable/ic_launcher_monochrome.xml b/app/src/release/res/drawable/ic_launcher_monochrome.xml
index 5a54380..0e81584 100644
--- a/app/src/release/res/drawable/ic_launcher_monochrome.xml
+++ b/app/src/release/res/drawable/ic_launcher_monochrome.xml
@@ -1,15 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Cloud Vault mark (monochrome/themed-icon variant): same padlock silhouette. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="108dp"
     android:height="108dp"
     android:viewportWidth="108"
     android:viewportHeight="108">
-    <group
-        android:scaleX="0.099"
-        android:scaleY="0.099"
-        android:translateX="24.3"
-        android:translateY="24.3">
+    <group>
+        <clip-path
+            android:pathData="M0,0h108v108h-108z"/>
         <path
-            android:fillColor="#ffffff"
-            android:pathData="M481.4,102.2c-3.7,-3.7 -8.1,-5.6 -13.1,-5.6L131.7,96.6c-5.1,0 -9.4,1.9 -13.1,5.6C114.9,105.9 113,110.2 113,115.3v224.4c0,16.7 3.3,33.4 9.8,49.8c6.5,16.5 14.6,31.1 24.3,43.8c9.6,12.8 21.1,25.2 34.5,37.2c13.3,12.1 25.7,22.1 37,30.1c11.3,8 23.1,15.5 35.4,22.6c12.3,7.1 21,11.9 26.2,14.5c5.2,2.5 9.3,4.5 12.4,5.8c2.3,1.2 4.9,1.8 7.6,1.8c2.7,0 5.3,-0.6 7.6,-1.8c3.1,-1.4 7.3,-3.3 12.4,-5.8c5.2,-2.5 13.9,-7.4 26.2,-14.5c12.3,-7.1 24.1,-14.7 35.4,-22.6c11.3,-8 23.6,-18 37,-30.1c13.3,-12.1 24.8,-24.5 34.5,-37.2c9.6,-12.8 17.7,-27.4 24.2,-43.8c6.5,-16.5 9.8,-33.1 9.8,-49.8L487.3,115.3C487,110.2 485.1,105.9 481.4,102.2zM438,341.8C438,423 300,493 300,493L300,144.6h138C438,144.6 438,260.6 438,341.8z" />
+            android:pathData="M37,52 V42 A17,17 0 0 1 71,42 V52"
+            android:strokeColor="#FF000000"
+            android:strokeWidth="9"
+            android:strokeLineCap="round"
+            android:fillColor="#00000000"/>
+        <path
+            android:pathData="M32,50 H76 A8,8 0 0 1 84,58 V82 A8,8 0 0 1 76,90 H32 A8,8 0 0 1 24,82 V58 A8,8 0 0 1 32,50 Z"
+            android:fillColor="#FF000000"/>
+        <path
+            android:pathData="M54,61 C57.87,61 61,64.13 61,68 C61,70.71 59.43,73.06 57.15,74.24 L60,84 H48 L50.85,74.24 C48.57,73.06 47,70.71 47,68 C47,64.13 50.13,61 54,61 Z"
+            android:fillColor="#00000000"/>
     </group>
 </vector>
diff --git a/app/src/release/res/values/strings_non_localized.xml b/app/src/release/res/values/strings_non_localized.xml
index 1e33b3b..7e1d38a 100644
--- a/app/src/release/res/values/strings_non_localized.xml
+++ b/app/src/release/res/values/strings_non_localized.xml
@@ -1,4 +1,4 @@
 <resources>
     <!--Common-->
-    <string name="app_name" translatable="false">Bitwarden</string>
+    <string name="app_name" translatable="false">Cloud Vault</string>
 </resources>
-- 
2.43.0

